src/subcommand/server.rs

```diff
@@ -2285,7 +2285,7 @@ mod tests {
     server.assert_response_csp(
       format!("/preview/{}", InscriptionId::from(txid)),
       StatusCode::OK,
-      "default-src 'unsafe-eval' 'unsafe-inline' data:",
+      "default-src 'self' 'unsafe-eval' 'unsafe-inline' data:",
```

Comment:
do we need 'self' in /preview/?

Reply:
No, because `/preview/` loads the response from `/content/`